src/EnergyPlus/HeatBalanceManager.cc

```diff
@@ -339,6 +340,13 @@ namespace HeatBalanceManager {
 		// Get the heat balance input at the beginning of the simulation only
 		if ( ManageHeatBalanceGetInputFlag ) {
 			GetHeatBalanceInput(); // Obtains heat balance related parameters from input file
+
+			// Surface octree setup
+			//  The surface octree holds live references to surfaces so it must be updated
+			//   if in the future surfaces are altered after this point
+			surfaceOctree.init( DataSurfaces::Surface ); // Set up surface octree
```

Comment:
@DeadParrot is there a reason to initialize the octree for every simulation? Shouldn't we only initialize it if we are beyond the number of surfaces crossover?